app/src/main/java/edu/uc/coffeens/stocktracker/dto/Stock.kt

```diff
@@ -12,14 +13,10 @@ data class Stock(
     @SerializedName("initial_price") var stockPrice: String
 ) {
 
-    //Serialization to map the fields from the JSON response to the vars I've put in place
-
-
     /**
      * Convert JSON to string for displaying on the search bar
      * @return information about a stock.
      */
-    override fun toString(): String {
-        return "$stockCompany: $stockTicker \$$stockPrice"
-    }
+    override fun toString(): String = "$stockCompany: $stockTicker \$$stockPrice"
+
```

Comment:
Kotlin shortcut allows functions to be put on one line https://developer.android.com/kotlin/style-guide#expression_functions